python/lsst/ts/tunablelaser/ascii.py

```diff
@@ -30,133 +30,10 @@
-    writer : `None` or `StreamWriter`
-    log : `logging.Logger`
-    """
-
-    def __init__(self, host, port, timeout=1) -> None:
-        self.timeout = timeout
-        self.host = host
-        self.port = port
-        self.reader = None
-        self.writer = None
-        self.log = logging.getLogger(__name__)
-        self.lock = asyncio.Lock()
-
-    @property
-    def connected(self) -> bool:
-        """Return True if a client is connected to this server.
-
-        Returns
-        -------
-        bool
-        """
-        return not (
-            self.reader is None
-            or self.writer is None
-            or self.reader.at_eof()
-            or self.writer.is_closing()
-        )
-
-    async def connect(self):
-        """Connect to the server."""
-        if not self.connected:
-            async with self.lock:
-                connect_task = asyncio.open_connection(self.host, self.port)
-                self.reader, self.writer = await asyncio.wait_for(
-                    connect_task, self.timeout
-                )
-
-    async def disconnect(self):
-        """Disconnect from the server.
-
-        Safe to call even if disconnected.
-        """
-        if self.writer is None:
-            return
-        try:
-            await tcpip.close_stream_writer(self.writer)
-        except Exception:
-            self.log.exception("disconnect failed, continuing")
-        finally:
-            self.writer = None
-            self.reader = None
-
-    async def send_command(self, message):
-        """Send a command to server and receive a reply.
-
-        Parameters
-        ----------
-        message : `str`
-            The command to send to the server.
-
-        Returns
-        -------
-        reply : `bytes`
-            The reply from the server.
-
-        Raises
-        ------
-        `RuntimeError`
-            Raised if the client is not connected.
-        """
-        if not self.connected:
-            raise RuntimeError("Client not connected.")
-        async with self.lock:
-            self.log.debug(f"message={message.encode('ascii')}")
-            message = message.encode("ascii")
-            self.writer.write(message)
-            await self.writer.drain()
-            reply = await self.reader.readuntil(b"\x03")
-            reply = self.parse_reply(reply)
-            self.log.debug(f"reply={reply.encode('ascii')}")
-            return reply
-
-    def parse_reply(self, reply):
-        """Return the parsed reply.
-
-        Parameters
-        ----------
-        reply : `bytes`
-            The reply from the server
-
-        Returns
-        -------
-        decoded_message : `str`
-            The decoded and boilerplate stripped string.
-        """
-        decoded_message = reply.decode("ascii").rstrip("nmC\r\n\x03")
-        if decoded_message.startswith("'''"):
-            raise Exception(decoded_message)
-        return decoded_message
-
 
 class AsciiRegister:
```

Comment:
Please rename the file to have a more descriptive name than `ascii.py`, for instance `ascii_register.py`.